### PR TITLE
Create nikhilveer.json

### PR DESCRIPTION
**Pull request by NikhilVeeram**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/nikhilveer.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A Portfolio / Landing Page for Project Information and Development(s)",
+    "repo": "https://github.com/NikhilVeeram/NikhilVeeram.github.io",
+    "owner": {
+        "username": "NikhilVeeram",
+        "email": "nikhilveeram@icloud.com",
+        "twitter": "<NikhilVeeramac5>"
+    },
+    "record": {
+        "CNAME": "<NikhilVeeram>.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/nikhilveer.json:10]:
```suggestion
        "CNAME": "NikhilVeeram.github.io"
```

File: domains/nikhilveer.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A Portfolio / Landing Page for Project Information and Development(s)",
+    "repo": "https://github.com/NikhilVeeram/NikhilVeeram.github.io",
+    "owner": {
+        "username": "NikhilVeeram",
+        "email": "nikhilveeram@icloud.com",
+        "twitter": "<NikhilVeeramac5>"
```
Comment (WilliamDavidHarrison) [inline domains/nikhilveer.json:7]:
```suggestion
        "twitter": "NikhilVeeramac5"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
I can't reach your website

**Review by wdhdev — CHANGES_REQUESTED** [review]
I can't reach your website

**Comment by WilliamDavidHarrison** [comment]
Unreachable website + no response.

**Pull request closed without merging**